capstone/src/main/java/com/google/sps/servlets/VoterServlet.java

```diff
@@ -45,7 +45,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
     requestDispatcher.forward(request, response);
   }
 
-  private JSONObject getJSONObject(URLConnection connection) throws IOException {
+  public JSONObject getJSONObject(URLConnection connection) throws IOException {
```

Comment:
Move the first 2 lines out of this method and pass a string in as a parameter instead of a URLConnection (since we don't want to test the API)

Reply:
Optional: rename this as 
```public VotingEvent createVotingEvent(String scannerLine){...}```
and move the voting event creation inside.
Then instead of testing that a JsonObject is created, you'd test the voting event is created.